zuul.d/base.yaml

```diff
@@ -17,14 +17,13 @@
       - ^PROJECT$
       - ^README.md$
       - ^renovate.json$
-      - ^tests/.*$
       - ^kuttl-test.yaml$
       - molecule/.*
       - molecule-requirements.txt
       - .github/workflows
       - docs/.*
       - contribute/.*
-      - tests
+      - tests?
```

Comment:
We run kuttl test in zuul, and the kuttl test cases are under test(s)/kuttl in most cases so we should not ignore that directory.

Reply:
Done